keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -714,12 +902,22 @@ message ProvisionerRevokeBucketAccessResponse {
 # Graduation Criteria
 
 ## Alpha
-
 - API is reviewed and accepted
-- Implement all COSI components to support Greenfield, Green/Brown Field, Brownfield and Static Driverless provisioning
-- Evaluate gaps, update KEP and conduct reviews for all design changes 
+- Design COSI APIs to support Greenfield, Green/Brown Field, Brownfield and Static Driverless provisioning
+- Design COSI APIs to support authentication using access/secret keys, and IAM.
+- Evaluate gaps, update KEP and conduct reviews for all design changes
 - Develop unit test cases to demonstrate that the above mentioned use cases work correctly
 
+## Alpha -\> Beta
```

Comment:
Should we also consider other important features like quota/restrictions on BucketClass and BucketAccessClass? If we want to tackle this as a parallel problem, we could add it to a "Related/Future Features" section.

Reply:
still open

Reply:
I have added quota/restriction feature for Buckets and BucketAccesses in the alpha -> beta section